tutordeck/server/static/js/logs.js

```diff
@@ -23,14 +23,21 @@ htmx.on("htmx:sseBeforeMessage", function (evt) {
 	const data = JSON.parse(evt.detail.data);
 	const command = data.command;
 
-	// This means a parallel command just started its execution
-	if (!executedNewCommand && data.thread_alive) {
-		ShowCancelCommandButton();
+	// This means a parallel command is executing
+	if (data.thread_alive) {
+		// Check if we are on the same page on which the actual command was executed
+		// Each page defines its own `onRelevantPage` function
```

Comment:
Isn't onRelevantPage the same across the pages, and its relevantCommands that differ across pages?